lib/jsdom/living/events/EventTarget-impl.js

```diff
@@ -24,8 +24,17 @@ class EventTargetImpl {
       return;
     }
 
-    if (capture === undefined) {
-      capture = false;
+    if (typeof options === "boolean") {
+      options = { capture: options };
+    } else {
+      if (!options) {
+        options = {};
+      }
+      options = {
+        capture: typeof options.capture === "boolean" ? options.capture : false,
```

Comment:
typeof tests are incorrect here. You need to do `capture: Boolean(options.capture)`.